packages/synapse-react-client/src/components/SynapseForm/SynapseFormWrapper.tsx

```diff
@@ -122,7 +122,7 @@ export class SynapseFormWrapper extends React.Component<
       }
 
       //for not new form we need to get the data
-      //and if it is submitted form we need to pull appropriave schema versions
+      //and if it is submitted form we need to pull appropriaves schema versions
```

Comment:
Is this supposed to say "appropriate"?

Reply:
Don't recall seeing/changing this. Fixed now